diesel_migrations/migrations_internals/src/lib.rs

```diff
@@ -348,17 +351,8 @@ where
 
     let run_in_transaction = migration
         .metadata()
-        .and_then(|m| m.get("run_in_transaction"));
-    let run_in_transaction = match run_in_transaction.as_ref().map(AsRef::as_ref) {
-        Some("true") => true,
-        Some("false") | None => false,
-        Some(_) => {
-            return Err(MigrationError::InvalidMetadata(
-                "run_in_transaction must be a boolean".into(),
-            )
-            .into())
-        }
-    };
+        .and_then(|m| m.get("run_in_transaction"))
+        .unwrap_or(Ok(false))?;
```

Comment:
@sgrif You set the default for `run_in_transaction` to false [here](https://github.com/diesel-rs/diesel/pull/1685/files#diff-ed7f6c154205d63bf776e00598c41eb9R338), is that the intended behaviour? Or would this possibly break migrations that rely on the previous behaviour of always running inside a transaction?